### PR TITLE
audio: allow enabling opus dtx

### PR DESCRIPTION
**Pull request by fippo**
from the js console. This is an expert option.

@alvestrand PTAL -- cc @minyuel

### CODE REVIEW

File: src/content/peerconnection/audio/js/main.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ const offerOptions = {
   voiceActivityDetection: false
 };
 
+// Enabling opus DTX is an expert option without GUI.
+let useDtx = false;
```
Comment (alvestrand) [inline src/content/peerconnection/audio/js/main.js:41]: The linter is anal.
let useDtx; useDtx = false will presumably pass. Or find the magic "lint, lint go away" cantrip.

Comment (fippo) [reply src/content/peerconnection/audio/js/main.js:41]: we've run into that before in bandwidth.js ;-)

**Comment by fippo** [comment]
updated to also allow changing ptime. Can separate but...

File: src/content/peerconnection/audio/js/main.js [hunk 2]
```diff
@@ -241,6 +248,20 @@ function setDefaultCodec(mLine, payload) {
   return newLine.join(' ');
 }
 
+// Change the ptime. For opus supported values are [10, 20, 40, 60].
+// eslint-disable-next-line no-unused-vars
```
Comment (alvestrand) [inline src/content/peerconnection/audio/js/main.js]: can you add "For experts at the console only" here too? And add ptime to the description?

**Review by alvestrand — APPROVED** [review]

File: src/content/peerconnection/audio/js/main.js (hunk 2 above)
Comment (fippo) [reply src/content/peerconnection/audio/js/main.js]: done. moved to the top too to increase visibility.

**Pull request merged**